### PR TITLE
Show full argument by click (+bugfix)

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️
| Breaks BC?    | ❌

image
image
image

---

Also fixed firing `click` event when selecting text

### CODE REVIEW

**Review by vjik — CHANGES_REQUESTED** [review]
Revert space here:

image

In released version:

image

File: templates/development.php [hunk 1]
```diff
@@ -249,13 +249,49 @@ class="copy-clipboard"
...
         for (var i = 0, imax = callStackItems.length; i < imax; ++i) {
-            refreshCallStackItemCode(callStackItems[i]);
+            let stackItem = callStackItems[i];
+            refreshCallStackItemCode(stackItem);
 
             // toggle code block visibility
-            callStackItems[i].getElementsByClassName('element-wrap')[0].addEventListener('click', function (event) {
+            stackItem.querySelector('.show-arguments-toggle')?.addEventListener('click', function (e) {
+                e.stopPropagation()
+                console.log('click')
```
Comment (samdark) [inline templates/development.php:258]:
```suggestion
```

File: templates/development.php [hunk 2]
```diff
@@ -249,13 +249,49 @@ class="copy-clipboard"
...
+                // function-arguments-wrap
+
```
Comment (samdark) [inline templates/development.php:260]:
```suggestion
```

Comment (samdark) [reply templates/development.php:260]: The comment doesn't seem to add to the context that is already there.

File: templates/development.php [hunk 3]
```diff
@@ -249,13 +249,49 @@ class="copy-clipboard"
...
+                stackItem.getElementsByClassName('function-arguments-wrap')[0].classList.toggle('hidden')
+            });
+
+            // toggle code block visibility
+            const arguments = stackItem.querySelector('.arguments');
+            arguments?.addEventListener('select', function (e) {
+                e.stopPropagation()
+                e.stopImmediatePropagation()
+
```
Comment (samdark) [inline templates/development.php:269]:
```suggestion
                e.stopImmediatePropagation()
```

**Review by samdark — CHANGES_REQUESTED** [review]

File: templates/development.php [hunk 4]
```diff
@@ -249,13 +249,47 @@ class="copy-clipboard"
...
 
         for (var i = 0, imax = callStackItems.length; i < imax; ++i) {
-            refreshCallStackItemCode(callStackItems[i]);
+            let stackItem = callStackItems[i];
+            refreshCallStackItemCode(stackItem);
 
             // toggle code block visibility
-            callStackItems[i].getElementsByClassName('element-wrap')[0].addEventListener('click', function (event) {
+            stackItem.querySelector('.show-arguments-toggle')?.addEventListener('click', function (e) {
+                e.stopPropagation()
```
Comment (samdark) [inline templates/development.php:257]: Either let's add `;` everywhere or remove it everywhere. I'm for adding it.

**Review by samdark — APPROVED** [review]

**Review by vjik — APPROVED** [review]

File: templates/development.php (hunk 4 above)
Comment (xepozz) [reply templates/development.php:257]: I'll add it in the last PR in the sequence

**Pull request merged**